module_auto_update/wizards/module_upgrade.py

```diff
@@ -8,6 +8,16 @@
 class ModuleUpgrade(models.TransientModel):
     _inherit = 'base.module.upgrade'
 
+    @api.model
+    def get_module_list(self):
```

Comment:
We have a problem currently in this method. It gets triggered when user presses "Uninstall" button in any addon. That's a problem because you want to uninstall an addon and it turns out to ask you to upgrade a bunch of them sometimes. It happens in v9 too. I think it's a good chance to patch it too, what do you think?